test/integration/crossPartition.spec.ts

```diff
@@ -340,6 +341,34 @@ describe("Cross Partition", function() {
       await executeQueryAndValidateResults({ query, options, expectedOrderIds: expectedOrderedIds });
     });
 
+    it("Validate DISTINCT Query", async function() {
+      // simple order by query in string format
+      const query = "SELECT DISTINCT VALUE r.spam3 FROM root r";
+      const options = {
+        enableCrossPartitionQuery: true,
+        maxItemCount: 2
+      };
+
+      const expectedOrderedIds = Array.from(new Set(documentDefinitions.map(r => r["spam3"])));
+
+      // validates the results size and order
+      await executeQueryAndValidateResults({ query, options, expectedOrderIds: expectedOrderedIds });
+    });
+
+    it("Validate DISTINCT OrderBy Query", async function() {
+      // simple order by query in string format
+      const query = "SELECT DISTINCT VALUE r.spam3 FROM root r order by r.spam3";
+      const options = {
+        enableCrossPartitionQuery: true,
+        maxItemCount: 2
+      };
+
+      const expectedOrderedIds = Array.from(new Set(documentDefinitions.map(r => r["spam3"])));
```

Comment:
The test code seems the same here as the unordered variant. Shouldn't the results differ?

Reply:
Yep! It was actually flaky. Sometimes they matched. Fixed